lib/galaxy/datatypes/binary.py

```diff
@@ -2239,6 +2239,126 @@ def sniff(self, dataset):
         return False
 
 
+class BafTar(CompressedArchive):
+    """
+    Base class for common behavior of tar files of directory-based raw file formats
+    >>> from galaxy.datatypes.sniff import get_test_fname
+    >>> fname = get_test_fname('brukerbaf.d.tar')
+    >>> BafTar().sniff(fname)
+    True
+    >>> fname = get_test_fname('test.fast5.tar')
+    >>> BafTar().sniff(fname)
+    False
+    """
+    edam_data = "data_2536"  # mass spectrometry data
+    edam_format = "format_3712"  # TODO: add more raw formats to EDAM?
+    file_ext = "brukerbaf.tar"
+
+    def get_signature_file(self):
+        return "analysis.baf"
+
+    def sniff(self, filename):
+        try:
```

Comment:
This is unnecessary, please remove the try/except around the sniff, that's caught outside. We really don't want to log anything here.